build of 1.2.3 fails on debian sid: cannot convert 'bool' to 'caret::VolumeFile*' in return

just thought to run by before looking into fixing myself ;)
```
[ 25%] Building CXX object Files/CMakeFiles/Files.dir/ImageFile.cxx.o
cd /build/connectome-workbench-1.2.3/obj-x86_64-linux-gnu/Files && /usr/bin/c++   -DCARET_OS_LINUX -DHAVE_FREETYPE -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -isystem /usr/include/qt4 -isystem /usr/include/qt4/QtGui -isystem /usr/include/qt4/QtCore -isystem /usr/include/qt4/QtNetwork -I/build/connectome-workbench-1.2.3/src/Annotations -I/build/connectome-workbench-1.2.3/src/Charting -I/build/connectome-workbench-1.2.3/src/Files -I/build/connectome-workbench-1.2.3/src/FilesBase -I/build/connectome-workbench-1.2.3/src/Cifti -I/build/connectome-workbench-1.2.3/src/Palette -I/build/connectome-workbench-1.2.3/src/Gifti -I/build/connectome-workbench-1.2.3/src/Nifti -I/build/connectome-workbench-1.2.3/src/Scenes -I/build/connectome-workbench-1.2.3/src/Xml -I/build/connectome-workbench-1.2.3/src/Common  -fopenmp -g -O2 -fdebug-prefix-map=/build/connectome-workbench-1.2.3=. -fstack-protector-strong -Wformat -Werror=format-security -DNDEBUG -Wdate-time -D_FORTIFY_SOURCE=2     -W -Wall -Werror=return-type -Werror=switch -Wunused-parameter -Wno-narrowing -Wno-unused-local-typedefs -o CMakeFiles/Files.dir/ImageFile.cxx.o -c /build/connectome-workbench-1.2.3/src/Files/ImageFile.cxx
/build/connectome-workbench-1.2.3/src/Files/ImageFile.cxx: In member function 'caret::VolumeFile* caret::ImageFile::convertToVolumeFile(caret::ImageFile::CONVERT_TO_VOLUME_COLOR_MODE, const std::vector<caret::ControlPoint3D>&, const caret::PaletteFile*, caret::AString&) const':
/build/connectome-workbench-1.2.3/src/Files/ImageFile.cxx:1423:16: error: cannot convert 'bool' to 'caret::VolumeFile*' in return
         return false;
                ^~~~~
Files/CMakeFiles/Files.dir/build.make:1625: recipe for target 'Files/CMakeFiles/Files.dir/ImageFile.cxx.o' failed
make[3]: *** [Files/CMakeFiles/Files.dir/ImageFile.cxx.o] Error 1
make[3]: Leaving directory '/build/connectome-workbench-1.2.3/obj-x86_64-linux-gnu'
CMakeFiles/Makefile2:748: recipe for target 'Files/CMakeFiles/Files.dir/all' failed
make[2]: *** [Files/CMakeFiles/Files.dir/all] Error 2
make[2]: Leaving directory '/build/connectome-workbench-1.2.3/obj-x86_64-linux-gnu'
Makefile:141: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/build/connectome-workbench-1.2.3/obj-x86_64-linux-gnu'
```
full buildlog   http://neuro.debian.net/_files/_buildlogs/connectome-workbench/1.2.3/connectome-workbench_1.2.3-1_amd64.build

--- a/src/Files/ImageFile.cxx
+++ b/src/Files/ImageFile.cxx
@@ -1420,7 +1420,7 @@ ImageFile::convertToVolumeFile(const CONVERT_TO_VOLUME_COLOR_MODE colorMode,
     if ((pixelNormalVector[2] < tinyValue)
         && (pixelNormalVector[2] > -tinyValue)) {
         errorMessageOut = "Control points need to be in a triangular shape; not a line";
-        return false;
+        return NULL;
     }
     
     if (pixelNormalVector[2] < 0.0) {

